web/app/js/components/Sidebar.jsx

```diff
@@ -202,7 +201,53 @@ class Sidebar extends React.Component {
               <Menu.Item><PrefixedLink to="/pods">Pods</PrefixedLink></Menu.Item>
               <Menu.Item><PrefixedLink to="/replicationcontrollers">Replication Controllers</PrefixedLink></Menu.Item>
             </Menu.SubMenu>
+            <Menu.Divider />
+          </Menu>
+
+          <Menu
+            className="sidebar-menu"
+            theme="dark"
+            mode="inline"
+            selectedKeys={[normalizedPath]}>
+            <Menu.Item className="sidebar-menu-item" key="/namespace-selector">
+              <Form layout="inline">
+                <Form.Item>
+                  <Select
+                    defaultValue="All Namespaces"
+                    dropdownMatchSelectWidth={true}
+                    onChange={this.handleNamespaceSelector}>
+                    {
+                      _.map(namespaces, ns => {
+                        return (
+                          <Select.Option value={ns.nsValue}>{ns.nsName}</Select.Option>
+                        );
+                      })
+                    }
+                  </Select>
+                </Form.Item>
+              </Form>
+            </Menu.Item>
 
+            {
+              _.map(_.keys(sidebarComponents).sort(), resourceName => {
+                return (
+                  <Menu.SubMenu
+                    className="sidebar-menu-item"
+                    key={resourceName}
+                    title={<span>{friendlyTitle(resourceName).plural}</span>}>
```

Comment:
sorry I didn't catch this before, but `title` tags contain text only and ignore tags within the element. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title

But also, unlike the pod names, I don't think there's any case where the full Resource Title will be cut off in the sidebar as is. So I think we should remove this `title=` entirely.

I can do that in a small branch of misc fixes I'm working on.

Reply:
Ah! thanks for catching that. I wasn't aware that the title prop was going to be the page title. I assumed it was something that antd used for displaying the `Menu.SubMenu`. Thanks for taking care of this!

Reply:
oh, blah, wrong link, I meant the attribute... *buries face*
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/title